docs/pages/build/ios-builds.md

```diff
@@ -0,0 +1,85 @@
+---
+title: iOS Builds
+---
+
+This page describes the process of building iOS projects with EAS Builds. Dive in if you're interested in the implementation details of our build service.
+
+## Build Process
+
+Let's take a closer look at the steps for building iOS projects with EAS Builds.
+
+### Local Steps
+
+The first phase is happening on your computer, Expo CLI is in charge of completing the following steps:
+
+1. Check if the git index is clean - this means if there aren't any uncommitted changes. If it's not clean an error is thrown.
+2. Determine the bundle identifier for the project.
+
+   If `expo.ios.bundleIdentifier` is specified in `app.json` but it doesn't match the bundle identifier in the Xcode project you're asked which one to use. If you decide to use the one from `app.json` we will automatically update it in the Xcode project.
+
+3. Prepare the credentials needed for the build.
+
+   Depending on the value of `builds.ios.PROFILE_NAME.credentialsSource`, the credentials are obtained from either the local `credentials.json` file or from the Expo servers. If the `auto` or `remote` mode is selected but no credentials exist yet, you're offered to generate them.
+
+4. Configure the Xcode project.
+
+   4.1. Ensure the correct bundle identifier and Apple Team ID are set.
+
+   4.2. Set the id of the provisioning profile resolved in the previous step.
+
+5. Create the tarball containing all your project sources - run `git archive --format=tar.gz --prefix project/ -o project.tar.gz HEAD`.
+6. Upload the project tarball to AWS S3 and send the build request to EAS Builds.
```

Comment:
```suggestion
6. Upload the project tarball to a private AWS S3 bucket and send the build request to EAS Builds.
```

or whatever we decide from the android section is a better substitute to emphasize secure storage of sources